scripts/install_dependencies.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eo pipefail
+
+if [[ -z ${SC_MACHINE_PATH+1} ]];
+then
+  CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+  source "${CURRENT_DIR}/set_vars.sh"
+  source "${CURRENT_DIR}/formats.sh"
+fi
+
+stage "Install dependencies"
+
+"${SC_MACHINE_PATH}/scripts/install_deps_ubuntu.sh" --dev
```

Comment:
I don't think `--dev` flag should be hardcoded. What if I want to install the runtime dependencies of the whole platform, without installing the dev ones? This is a usecase for final stages of docker containers, for example.